test/e2e/e2e.test.js

```diff
@@ -0,0 +1,172 @@
+    const { apiHost, apiPort } = global.ipfs
+    const apiMultiaddr = `/ip4/${apiHost}/tcp/${apiPort}`
+    // point webui at real api
+    await page.evaluate((apiMultiaddr) => localStorage.setItem('ipfsApi', apiMultiaddr), apiMultiaddr)
+    // apply settings
+    await page.goto(landingPage)
+  })
+
+  it('should have Status menu item', async () => {
+    // this is just a basic smoke-test to tell if page loads at all
+    await expect(page).toMatch('Status')
+  })
+
+  it('should inform it is sucessfully connected to IPFS', async () => {
+    // confirm webui thinks it is connected to node
+    await expect(page).toMatch('Connected to IPFS')
+  })
+
+  it('should display Peer ID of real IPFS node', async () => {
+    // confirm webui is actually connected to expected node :^)
+    const { id } = await ipfs.id()
+    await expect(page).toMatch(id)
+  })
+})
+
+describe('Navigation menu', () => {
+  beforeAll(async () => {
+    await page.goto(landingPage)
+  })
+
+  it('should work for Status page', async () => {
+    await expect(page).toMatch('Status')
+    await page.click('a[href="#/"]')
+    await waitForTitle('Status - IPFS')
+  })
+
+  it('should work for Files page', async () => {
+    await expect(page).toMatch('Files')
+    await page.click('a[href="#/files"]')
+    await waitForTitle('/ - Files - IPFS')
+  })
+
+  it('should work for Explore page', async () => {
+    await expect(page).toMatch('Explore')
+    await page.click('a[href="#/explore"]')
+    await waitForTitle('Explore - IPLD')
+  })
+
+  it('should work for Peers page', async () => {
+    await expect(page).toMatch('Peers')
+    await page.click('a[href="#/peers"]')
+    await waitForTitle('Peers - IPFS')
+  })
+
+  it('should work for Settings page', async () => {
+    await expect(page).toMatch('Settings')
+    await page.click('a[href="#/settings"]')
+    await waitForTitle('Settings - IPFS')
+  })
+})
+
+describe('Files screen', () => {
+  beforeAll(async () => {
+    await page.goto(landingPage + '#/files')
+  })
+
+  const button = 'button[id="add-button"]'
+
+  it('should have the active Add menu', async () => {
+    await page.waitForSelector(button, { visible: true })
+    await page.click(button)
+    await page.waitForSelector('#add-file', { visible: true })
+    await expect(page).toMatch('File')
+    await expect(page).toMatch('Folder')
+    await expect(page).toMatch('From IPFS')
+    await expect(page).toMatch('New folder')
+    await page.click(button)
+  })
+
+  it('should allow for a successful import of a single file', async () => {
+    await page.waitForSelector(button, { visible: true })
+    await page.click(button)
+    await page.waitForSelector('#add-file', { visible: true })
+
+    const [fileChooser] = await Promise.all([
+      page.waitForFileChooser(),
+      page.click('a[id="add-file"]'), // menu button that triggers file selection
+    ])
+
+    // lets add a static text file from the root of this repo
+    const filename = 'LICENSE'
```

Comment:
Seems like this would break if we change the location of the tests, eg into a subdir. Maybe just commit a test file to use explicitly for this purpose?